src/engine/renderer/tr_image.cpp

```diff
@@ -1611,6 +1611,58 @@ static const imageExtToLoaderMap_t imageLoaders[] =
 
 static int                   numImageLoaders = ARRAY_LEN( imageLoaders );
 
+/*
+=================
+R_FindAltImage
```

Comment:
maybe, naming is part of "work in progress"

Reply:
I renamed it to `R_FindImageLoader` because it's its primary purpose, a loader is found because an alternative image is found for this loader, and the function returns the loader index. It also fills the `prefix` blank if the image loader needs this prefix.

Since this function checks if an image basename has a loader, it may be abused to check if a file exists with a given basename. This is what the DarkPlaces extra texture lookup code does: it computes a brand new image basename and checks if it exists.